### ISSUE TITLE
If ES backend is down, show error message, not just wait 

### ISSUE THREAD

**Issue by alexanderpanchenko** [uhh-lt/cam#45]
Rather fail quickly: otherwise it is annoying to wait and never get the results. If ES is N/A we need to show a different message than just 'Loading ...' forever.

**Comment by mschild** [comment]
No problem if the ES sents a 503 (not available) to the backend. It just needs to be propagated further to the frontend probably.

By the way, did you manage to deploy the backend with the small change (threaded=true)?
Is there a way I can access the place where the backend and the frontend is deployed?

**Comment by alexanderpanchenko** [comment]
sorry, not yet. i've sent you to email credentials.

**Comment by mschild** [comment]
No problem.
Alright, thanks!

**Comment by mschild** [comment]
Yes, with a bit of initial information given by Julian, I now know how to deploy the parts of the system.
So #46  and #45 are available now.
